Composer/packages/adaptive-form/src/components/fields/IntellisenseFields.tsx

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { FieldProps } from '@bfc/extension-client';
+import { Intellisense } from '@bfc/intellisense';
+import React from 'react';
+
+import { getIntellisenseUrl } from '../../utils/getIntellisenseUrl';
+import { ExpressionSwitchWindow } from '../ExpressionSwitchWindow';
+
+import { JsonField } from './JsonField';
+import { NumberField } from './NumberField';
+import { StringField } from './StringField';
+
+export const IntellisenseTextField: React.FC<FieldProps<string>> = function IntellisenseTextField(props) {
+  const { id, value = '', onChange, uiOptions, focused: defaultFocused } = props;
+
+  const completionListOverrideResolver = (value: string) => {
+    if (value === '') {
```

Comment:
nit: ```value === '' ? <ExpressionSwitchWindow type={'String'} onSwitchToExpression={() => onChange('=')} />: null```

Reply:
Should null be returned? or would </> be better (empty fragment)?

Reply:
@GeoffCoxMSFT why fragment? null is like don't render anything.

Reply:
hatpick: done. @GeoffCoxMSFT, this is an override, null means no override. </> would show an empty Intellisense panel, which is not what we want.